driver-pulsar/deploy/ssd/templates/bookkeeper.conf

```diff
@@ -292,15 +296,15 @@ auditorPeriodicBookieCheckInterval=86400
 # number of threads that should handle write requests. if zero, the writes would
 # be handled by netty threads directly.
 numAddWorkerThreads=8
```

Comment:
By default the `numAddWorkerThreads=8` should actually be 0 so we directly write from the IO threads